googlemock/cmake/Config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
+check_required_components("@PROJECT_NAME@")
```

Comment:
> sometimes Visual Studio

Looks like it's not possible out-of-the-box: http://stackoverflow.com/questions/2254269/visual-c-setting-to-insert-a-newline-at-end-of-file

Let me know if you'll find some automatic-control solution, I'll add this info to the docs for other developers.